### PR TITLE
STORM-977 Correct log prefix (-15) when sending -9 signal to worker

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

File: storm-core/src/clj/backtype/storm/daemon/supervisor.clj [hunk 1]
```diff
@@ -262,7 +262,7 @@
       (psim/kill-process thread-pid))
     (doseq [pid pids]
       (if as-user
-        (worker-launcher-and-wait conf user ["signal" pid "9"] :log-prefix (str "kill -15 " pid))
+        (worker-launcher-and-wait conf user ["signal" pid "9"] :log-prefix (str "kill -9 " pid))
```
Comment (caofangkun) [inline storm-core/src/clj/backtype/storm/daemon/supervisor.clj]: I think should change the signal value but not the ```log-prefix```
should be 
 ```(worker-launcher-and-wait conf user ["signal" pid "15"] :log-prefix (str "kill -15 " pid))```

**Comment by caofangkun** [comment]
The travis ci build failed does not make sense ,
Does ``` export STORM_TEST_TIMEOUT_MS=100000``` not long enough ?

File: storm-core/src/clj/backtype/storm/daemon/supervisor.clj (hunk 1 above)
Comment (d2r) [inline storm-core/src/clj/backtype/storm/daemon/supervisor.clj]: @caofangkun You are correct.  The JIRA title is wrong.

We want to send the SIGTERM, then wait, then send SIGKILL.

**Comment by HeartSaVioR** [comment]
@caofangkun Build issue is resolved now, but old PRs still need to be upmerged to take effect.

**Comment by d2r** [comment]
+1

**Pull request merged**